app/code/Magento/Quote/Model/ResourceModel/Quote.php

```diff
@@ -167,7 +167,7 @@ public function getReservedOrderId($quote)
     {
         return $this->sequenceManager->getSequence(
             \Magento\Sales\Model\Order::ENTITY,
-            $quote->getStore()->getGroup()->getDefaultStoreId()
+            $quote->getStore()->getStoreId()
```

Comment:
I see that $quote has method getStoreId(). Isn't it better to use it?

Reply:
Thank you for the review.

I'd say that's better indeed, but it shouldn't matter nonetheless. If you want it to use `$quote->getStoreId()` I can change that.